src/Engine/Systems/Subsystems/Physics/OrbitPointGen.hpp

```
@@ -0,0 +1,265 @@
+       @return     3x3 rotation matrix mapping perifocal (PQW) -> ECI (IJK)
+    */
+    inline glm::dmat3 PerifocalToECI(const COE::Elements &coe) {
+        using enum Physics::OrbitGeometry;
+        using enum Physics::OrbitInclination;
+
+        double omega = coe.omega;
+        double argp = coe.argp;
+        double incl = coe.incl;
+
+        // Degenerate substitutions per Vallado
+        if (std::isnan(omega)) omega = 0.0;     // Equatorial orbit: RAAN undefined, set to 0
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)